app/src/main/java/com/github/se/eventradar/model/event/Event.kt

```diff
@@ -11,7 +11,7 @@ data class Event(
     var location: Location,
     var description: String,
     var ticket: EventTicket,
-    var contact: String,
+    var mainOrganiser: String,
```

Comment:
**Question:** Thank you for implementing the change. However, I've noticed that there are no updates in the file EventDetailsViewModel.kt, which I expected might need modifications due to this property change. Could you clarify if changes are needed there as well?

Reply:
I don't think so, because only the data layer interacts with this class.

Reply:
**Solved:** The refactoring of contact to mainOrganiser in the event data class did not automatically update the contact property in the EventUiState since it's a different data class. However, the contact property in EventUiState should also be updated to mainOrganiser to maintain consistency across both classes.